api/models/dishesModel.js

```diff
@@ -0,0 +1,45 @@
+const db = require('../../data/dbConfig.js')
+
+module.exports = {
+    find,
+    findById,
+    findRecipesById,
+    add,
+    update,
+    remove
+}
+
+function find() {
+    return db('dishes')
+}
+
+function findById(id) {
+    return db('dishes')
+        .where({ id })

```

Comment:
```
async function findById(id){
  const dish = await db('dishes').where({ id }).first();
  dish.recipes = await findRecipesById(id);
  return dish;
}
```
I can't think of a reason why we would want a dish by id and not have the recipes attached to it, this would save an api call from the front end. There's a few different ways to achieve the same result here, this is just the pattern I've fallen into.